src/FGFDMExec.h

```diff
@@ -123,7 +123,7 @@ CLASS DOCUMENTATION
     file can be supplied to the stub program. Scripting (see FGScript) provides
     a way to supply command inputs to the simulation:
 
-    @code{.cpp}
+    @code
```

Comment:
Please don't remove the `{.cpp}` elements
```suggestion
    @code{.cpp}
```